src/patternfly/components/ModalBox/styles.scss

```diff
@@ -3,6 +3,8 @@
 .pf-c-modal-box {
   // Component variables
   --pf-c-modal-box--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-modal-box--BorderSize: var(--pf-global--BorderWidth--sm);
```

Comment:
We don't need to change the modal box. This approach is for elements that have a hover or focus state that doubles the size of the border. 

Whenever the border is a border, lets use a border :)

Reply:
To recap what is noted in [this presentation](https://www.youtube.com/watch?v=Wq2-cZ-vZBc), the suggestion is to use this method for any actionable or overlay element where the border is replaced with a box shadow or background color. I do think this is a rule we should add to the a11y guide and something we should follow.